signing/src/test/java/tech/pegasys/web3signer/signing/BlsBKeystoreBulkLoaderTest.java

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright 2022 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.web3signer.signing;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import tech.pegasys.teku.bls.BLSKeyPair;
+import tech.pegasys.web3signer.BLSTestUtil;
+import tech.pegasys.web3signer.KeystoreUtil;
+
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Collection;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+
+class BlsBKeystoreBulkLoaderTest {
```

Comment:
These tests take 13 secs on my machine. Will the tests be significantly faster if reuse the @TempDirs or is the time crypto/keystore related?

Reply:
Takes 4s on my machine which I think is ok. It's probably the decrypt taking a long time?

Reply:
Since this is only for tests we can make it more insecure as well. Perhaps use the settings that eth2-val-tools uses in their insecure mode?

Reply:
Reduced the number of pbkdf iterations to 2 as this is a test. Only takes 1.5s on my machine now. Hopefully it similarly faster on your machine now.